library/network_connections.py

```diff
@@ -222,31 +222,29 @@ def link_infos(cls, refresh=False):
         return linkinfos
 
     @classmethod
-    def link_info_find(cls, refresh=False, mac=None, ifname=None):
-        if mac is not None:
+    def link_info_find(cls, mac=None, ifname=None):
+        if mac:
```

Comment:
note that `""`, `[]`, and `{}` all evaluate to `False` in a boolean context - so if `mac == ""` the `if mac:` will evaluate to `False` - not sure if that is allowed

Reply:
`mac` should be in string format, the `ArgValidatorMac` already rules out the possibility of the empty string in this case.